### PR TITLE
feat(orchestrator): add Orchestrator plugins into container image

### PR DESCRIPTION
**Pull request by mareklibra**
## Description

Adds Orchestrator dynamic plugins into `Showcase` container image. The plugins are `disabled` by default.

## Which issue(s) does this PR fix

- Fixes [FLPATH-871](https://issues.redhat.com//browse/FLPATH-871)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins.default.yaml [hunk 1]
```diff
@@ -426,6 +426,32 @@ plugins:
...
+              - path: /orchestrator
+                importName: OrchestratorPage
+                module: OrchestratorPlugin
+                menuItem:
+                  icon: orchestratorIcon
+                  text: Orchestrator                  
+  - package: ./dynamic-plugins/dist/backstage-plugin-orchestrator-backend-dynamic
+    disabled: true
+    pluginConfig:
+      orchestrator:
```
Comment (mareklibra) [inline dynamic-plugins.default.yaml:450]: Taken from the plugin itself but I am not sure we want this here.

**Comment by pkliczewski** [comment]
@christophe-f fyi

File: dynamic-plugins.default.yaml (hunk 1 above)
Comment (mareklibra) [reply dynamic-plugins.default.yaml:450]: changed

File: dynamic-plugins.default.yaml [hunk 2]
```diff
@@ -426,6 +426,33 @@ plugins:
...
+                module: OrchestratorPlugin
+                importName: OrchestratorIcon
+            dynamicRoutes:
+              - path: /orchestrator
+                importName: OrchestratorPage
+                module: OrchestratorPlugin
+                menuItem:
+                  icon: orchestratorIcon
+                  text: Orchestrator                  
+  - package: ./dynamic-plugins/dist/backstage-plugin-orchestrator-backend-dynamic
```
Comment (masayag) [inline dynamic-plugins.default.yaml:447]: pls change the path to `./dynamic-plugins/dist/backstage-plugin-orchestrator-backend-dynamic`

File: dynamic-plugins.default.yaml [hunk 3]
```diff
@@ -426,6 +426,33 @@ plugins:
                     - isKind: resource
                     - isType: kubernetes-cluster
 
+  # Group: Orchestrator
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator
```
Comment (masayag) [inline dynamic-plugins.default.yaml:430]: the path inside the image is `./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator-backend-dynamic`

File: dynamic-plugins.default.yaml (hunk 2 above)
Comment (masayag) [inline dynamic-plugins.default.yaml:447]: the path inside the image is `./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator-backend-dynamic`

**Comment by mareklibra** [comment]
Rebased due to conflicts

**Comment by nickboldt** [comment]
https://issues.redhat.com//browse/FLPATH-871 was closed won't do, so this can be closed otoo.

**Pull request closed without merging**